fem/bilininteg.hpp

```diff
@@ -1634,16 +1645,18 @@ class DiffusionIntegrator: public BilinearFormIntegrator
 #endif
    Coefficient *Q;
    MatrixCoefficient *MQ;
-
+   // PA extension
+   DofToQuad *maps;
```

Comment:
"maps" is a pretty general name.  Perhaps dtq_maps?

Reply:
"dtq" has a very vulgar connotation in French, but other than this, more self documented names is always better.

Reply:
I'll have to look that one up when I get home.